src/parse.rs

```diff
@@ -226,7 +226,7 @@ impl MidiByteStreamParser {
                 }
                 MidiParserState::PitchBendFirstByteRecvd(channel, byte1) => {
                     self.state = MidiParserState::PitchBendRecvd(channel);
-                    Some(MidiMessage::PitchBendChange(channel, (byte1, byte).into()))
+                    Some(MidiMessage::PitchBendChange(channel, (byte, byte1).into()))
```

Comment:
Maybe you could rename the byte values to `lsb` and `msb`? That would make issues like the one you fixed stand out more.